3rdparty/LICENSE.pytest-lazy-fixtures

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
```

Comment:
Given `pytest-lazy-fixtures` bundles the license file in the wheel and conda packages, do we need to bundle the license file here or could it be dropped?

Reply:
Thanks @jakirkham !

All files under '3rdparty' folder is actually optional and we added the folder for tracking licenses internally.

Since 'LICENSE-3rdparty.md' file on the root folder covers the list of 3rdparty libraries, do you think it is better to remove whole '3rdparty' folder instead of maintaining the folder?

Reply:
That's a good idea. Let's follow up after we finish the devendoring work: https://github.com/rapidsai/cucim/pull/753

Since some of those dependencies are statically linked now, we do need to include their licenses. Though it is possible after devendoring that we can substantially reduce or drop licenses from this list

With Python only dependencies, we could clean those up before devendoring, but it might make sense to wait until we can do a more thorough cleanup